### DISCUSSION TITLE
Qual melhor framework JS para substituir o PHP em sites simples?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1296] [Perguntas e Respostas]
Boa tarde, tenho uma dúvida a respeito de trocar o PHP do backend de site simples (SQL, Rotas e etc). Queria deixar tudo com JS. O que vocês me recomendam?
Algum framework bom para isso?

**Comment by Ian-FR** [comment]
Dá para usar o **ExpressJS** (http://expressjs.com/) que é em nodeJS, não usei em projetos em produção ainda mas pretendo usar para Backend de um app feito com o Flutter futuramente.

**Comment by luizbills** [comment]
Depende de como você usa o PHP. Se você costuma usar com Laravel ou outro MVC framework, então recomendo você estudar algum framework MVC em nodejs, como o [Sails.js](https://sailsjs.com/). Agora, se você é costuma apenas instalar alguns pacotes que cuidam das rotas, do banco de dados, etc... ai vai com ExpressJS e algum outro pacote para cuidar do BD, como o Sequelize.

**Comment by isacmoura** [comment]
Se for pra renderizar as informações eu uso o Express com ejs, e já vi usarem o handlebars
